eventmesh-protocol-plugin/eventmesh-protocol-http/src/main/java/org/apache/eventmesh/protocol/http/HttpProtocolAdaptor.java

```diff
@@ -95,8 +95,11 @@ public ProtocolTransportObject fromCloudEvent(CloudEvent cloudEvent) throws Prot
         }
         httpEventWrapper.setSysHeaderMap(sysHeaderMap);
         // ce data
-        Map<String, Object> dataContentMap = JsonUtils.deserialize(new String(cloudEvent.getData().toBytes(), Constants.DEFAULT_CHARSET),
+        String jsonStr = new String(cloudEvent.getData().toBytes(), Constants.DEFAULT_CHARSET);
```

Comment:
`cloudEvent.getData()` may due NPE, please check this return value, note the `jsonStr`